packages/appeals-service-api/src/services/appeal-user.service.js

```diff
@@ -0,0 +1,19 @@
+const logger = require('../lib/logger');
+const { AppealUserRepository } = require('../repositories/sql/appeal-user-repository');
+const ApiError = require('../errors/apiError');
+
+const appealUserRepository = new AppealUserRepository();
+
+const getUserByEmail = async (email) => {
+	try {
+		const user = await appealUserRepository.getByEmail(email);
+		return user;
+	} catch (err) {
+		logger.error(err);
+		throw ApiError.appealsCaseDataNotFound();
```

Comment:
I think at this point this would be an unknown error of some kind - assuming `getByEmail` returns null or similar if not found.